### DISCUSSION TITLE
Shut down SEB?

### DISCUSSION THREAD

**Discussion by AliceSFC** [SafeExamBrowser/seb-win-refactoring#995] [Q&A]
Hello
Teachers at our school regularly work with SEB. We find that underlying SEB often continues to run even though students are no longer in a test resulting in a number of problems. Regularly, students cannot enter a new test, but closing SEB via task management often solves the problem. Recently, this has become more common, causing teachers to lose a lot of valuable time during their lessons. 
It also regularly happens that students cannot shut down their laptops correctly. They then get the message that no power management options are available.
Is there a quick, easy solution to this problem? 
We notice that due to the problems, teachers are using SEB less, which we regret.
Thanks in advance
Alice

**Comment by dbuechel** [comment]
With which SEB version(s) are you experiencing these issues?
